series-formelles.tex

```diff
@@ -1730,8 +1730,7 @@ \subsection{Differential calculus}
 \begin{ex} \label{ex:trees-vertebrates}
 To illustrate the combinatorial differential calculus,
 consider the equation satisfied by the species $A$ of rooted trees:
-\[ A = X \cdot e^A. \] The operation of ``pointing'' \trans{is a
-  derivation}{est une d\'erivation}:
+\[ A = X \cdot e^A. \] The ``pointing'' operation is differentiation:
```

Comment:
Translating this as "...operation is differentiation" does not make sense, since pointing is *not* differentiation, and Joyal surely was aware of the distinction.  Does the indefinite article 'une' really not carry any force?  Perhaps the point is that pointing is a derivative-like operator, seeing as it follows this analogue to the product rule?

Reply:
"Le pointage est une dérivation" does have a looser meaning than "le pointage est la dérivation", and it could be taken to mean that it looks like differentiation.